app/javascript/components/froggo-team-form.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <span>Nombre del equipo:</span><br><br>
+    <input type="text" v-model="team_name"><br><br>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'v-model' should be on a new line                                                 vue/max-attributes-per-line